### DISCUSSION TITLE
Align part of the element to the right

### DISCUSSION THREAD

**Discussion by ollosh** [davatorium/rofi#1610] [Q&A] [answered]
Is it possible to align part of the element with -dmenu to the right. I'm now using a hacky way of doing it which is not perfect and takes a lot of useless calculation for a non monospaced font.
![image](https://user-images.githubusercontent.com/12979324/161378625-ae71bbae-329d-4fab-826d-fb0296537f96.png)

Is there a way to do sth like in this pr for combi-menu https://github.com/davatorium/rofi/pull/1571

Or if I can't do it that way, is there a way to position element-icon to the right?

**Comment by DaveDavenport** [comment, marked as answer]
element icon is easy to move to the right, change the packing order of the `element-text` and `element-icon` widget in the `element` container in your theme.

To align text right you could use the ´tab-stop'  support in the latest release. that would work for above image.
See:
![rofi-2022-04-02-1254-00000](https://user-images.githubusercontent.com/406245/161380098-b61a8fd7-bef8-4284-a623-912d075d5004.png)

**Reply by ollosh** [reply]
Yes in the meantime I found a way to do it with icons, just have some issues with null bytes in my python script and will post my solution once I'm done. 
However, since you have mentioned 'tab-stop' from the last release, does it work with -dmenu as well? If so, would you mind sharing the rofi dmenu command that illustrates how it should be used?

**Reply by DaveDavenport** [reply]
yes, it will work with dmenu, as it is a theme option.

**Reply by ollosh** [reply]
Ah to answer my question it does work with -dmenu as well! I was looking at the pr of the implementation and was confused why `tab-stops: 500px` wasn't working, but just wrapping it in square brackets worked, so: `tab-stops: [500px]` works. 
Great stuff!

**Reply by DaveDavenport** [reply]
yes, it takes a list as you can specify multiple stops.
